lib/classes/Variables.test.js

```diff
@@ -1359,6 +1360,55 @@ module.exports = {
         .populateProperty(property)
         .should.eventually.eql('my stage is dev');
     });
+
+    it('should support ${file(...)} syntax', () => {
+      return runServerless({
+        fixture: 'variables',
+        cliArgs: ['print'],
+      }).then(result => {
```

Comment:
Let's stick to async functions (so `await` syntax)

Reply:
👍  I changed it (I initially copied the style I found in the existing code).